bottlecap/src/metrics/aggregator.rs

```diff
@@ -604,8 +604,8 @@ mod tests {
 
     #[test]
     fn consume_distributions_batch_bytes() {
-        let single_proto_size = 104;
-        let max_bytes = 250;
+        let single_proto_size = 135;
+        let max_bytes = 300;
```

Comment:
max_bytes should not be changed, unless it moves the threshold enough to prevent expected batching

Reply:
This was needed because or else, batches would mismatch, since the size change, 250 was max 2 batches of 104, now that the number is 135, we need at least 270 for proper batching